sdk/src/main/scala/com/horizen/certificatesubmitter/CertificateSubmitter.scala

```diff
@@ -748,4 +554,11 @@ object CertificateSubmitterRef {
             mainchainChannel: MainchainNodeChannel)
            (implicit system: ActorSystem, ec: ExecutionContext): ActorRef =
     system.actorOf(props(settings, sidechainNodeViewHolderRef, params, mainchainChannel).withMailbox("akka.actor.deployment.submitter-prio-mailbox"), name)
+
+  object TypeOfCircuit extends Enumeration {
+    type TypeOfCircuit = Value
+
+    val NaiveThresholdSignatureCircuit, NaiveThresholdSignatureCircuitWithKeyRotation = Value
+  }
```

Comment:
I assume that this Enum is going to be used in multiple places, for example in SidechainApp.
better to move it to the separate file in `com.horizen.cryptolibprovider` package.